diff --git a/node_modules/@openid4vc/utils/dist/index.js b/node_modules/@openid4vc/utils/dist/index.js
index 2b6653e..236ec93 100644
--- a/node_modules/@openid4vc/utils/dist/index.js
+++ b/node_modules/@openid4vc/utils/dist/index.js
@@ -343,6 +343,9 @@ Cause: ${cause?.message}`);
 var defaultFetcher = fetch;
 function createFetcher(fetcher = defaultFetcher) {
   return (input, init, ...args) => {
+    console.log("############### input #################", input);
+    console.log("############### init #################", init);
+    console.log("############### args #################", args);
     return fetcher(
       input,
       init ? {
