diff --git a/node_modules/@openid4vc/oauth2/dist/index.js b/node_modules/@openid4vc/oauth2/dist/index.js
index 51abb78..a2a6941 100644
--- a/node_modules/@openid4vc/oauth2/dist/index.js
+++ b/node_modules/@openid4vc/oauth2/dist/index.js
@@ -956,21 +956,27 @@ ${formattedError}`;
 // src/metadata/fetch-well-known-metadata.ts
 async function fetchWellKnownMetadata(wellKnownMetadataUrl, schema, fetch) {
   const fetcher = (0, import_utils10.createZodFetcher)(fetch);
-  const { result, response } = await fetcher(schema, import_utils10.ContentType.Json, wellKnownMetadataUrl);
-  if (response.status === 404) {
+  try{
+    const { result, response } = await fetcher(schema, import_utils10.ContentType.Json, wellKnownMetadataUrl);
+
+    if (response.status === 404) {
+      return null;
+    }
+    if (!response.ok) {
+      throw new import_utils11.InvalidFetchResponseError(
+        `Fetching well known metadata from '${wellKnownMetadataUrl}' resulted in an unsuccessfull response with status '${response.status}'.`,
+        await response.clone().text(),
+        response
+      );
+    }
+    if (!result?.success) {
+      throw new ValidationError(`Validation of metadata from '${wellKnownMetadataUrl}' failed`, result?.error);
+    }
+    return result.data;
+  }catch(e){
+    console.log('Changed By JS-SOFT: Error can probably be ignored', e);
     return null;
   }
-  if (!response.ok) {
-    throw new import_utils11.InvalidFetchResponseError(
-      `Fetching well known metadata from '${wellKnownMetadataUrl}' resulted in an unsuccessful response with status '${response.status}'.`,
-      await response.clone().text(),
-      response
-    );
-  }
-  if (!result?.success) {
-    throw new ValidationError(`Validation of metadata from '${wellKnownMetadataUrl}' failed`, result?.error);
-  }
-  return result.data;
 }
 
 // src/metadata/authorization-server/z-authorization-server-metadata.ts
