diff --git a/node_modules/@openid4vc/openid4vp/dist/index.js b/node_modules/@openid4vc/openid4vp/dist/index.js
index 575e4fd..78bdf6b 100644
--- a/node_modules/@openid4vc/openid4vp/dist/index.js
+++ b/node_modules/@openid4vc/openid4vp/dist/index.js
@@ -599,10 +599,12 @@ async function validateOpenid4vpClientId(options, parserConfig) {
       if (!isOpenid4vpAuthorizationRequestDcApi(authorizationRequestPayload)) {
         const uri = authorizationRequestPayload.redirect_uri ?? authorizationRequestPayload.response_uri;
         if (!uri || new import_utils5.URL(uri).hostname !== clientIdIdentifier) {
-          throw new import_oauth23.Oauth2ServerErrorResponseError({
-            error: import_oauth23.Oauth2ErrorCodes.InvalidRequest,
-            error_description: "Invalid client identifier. The fully qualified domain name of the redirect_uri value MUST match the Client Identifier without the prefix x509_san_dns."
-          });
+          if(!uri || new import_utils5.URL(uri).hostname.toLowerCase() !== clientIdIdentifier.toLowerCase()){
+            throw new import_oauth23.Oauth2ServerErrorResponseError({
+              error: import_oauth23.Oauth2ErrorCodes.InvalidRequest,
+              error_description: "Invalid client identifier. The fully qualified domain name of the redirect_uri value MUST match the Client Identifier without the prefix x509_san_dns."
+            });
+          }
         }
       }
     } else if (clientIdPrefix === "x509_san_uri") {
